### PR TITLE
add Qwen custom add model interface

### LINKED ISSUE #8438: Request to add Qwen custom add model interface
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Aliyun platform has a large number of other models. At present, only oneapi can be used to add other models. I hope to add a custom model interface to facilitate adding other models.

### 2. Additional context or comments

_No response_

### 3. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

### PR DESCRIPTION
**Pull request by AAEE86**
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [x] Please open an issue before creating a PR or link to an existing issue
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. Close issue syntax: `Fixes #<issue number>`, see [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for more details.

Fixes #8438

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [x] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Comment by AAEE86** [comment]
@Laipz8200 Is it correct to make modifications according to the md document and other LLM files?

**Comment by AAEE86** [comment]
Quicker_20240920_104851

File: api/core/model_runtime/model_providers/tongyi/llm/llm.py [hunk 1]
```diff
@@ -520,3 +520,84 @@ def _invoke_error_mapping(self) -> dict[type[InvokeError], list[type[Exception]]
                 UnsupportedHTTPMethod,
             ],
         }
+
+    def get_customizable_model_schema(self, model: str, credentials: dict) -> AIModelEntity | None:
+        """
+        用于定义自定义模型的 schema
```
Comment (laipz8200) [inline api/core/model_runtime/model_providers/tongyi/llm/llm.py]: Please comment on your code in English.

**Review by laipz8200 — CHANGES_REQUESTED** [review]

**Comment by laipz8200** [comment]
Hi @AAEE86, Please take a look at my comments above. You should run the `/dev/reformat` script before your commit.

**Comment by AAEE86** [comment]
> Hi @AAEE86, Please take a look at my comments above. You should run the `/dev/reformat` script before your commit.

pls merge.

**Review by crazywoola — APPROVED** [review]

**Pull request merged**